wp-modules/editor/model.php

```diff
@@ -125,7 +125,7 @@ function save_metadata_to_pattern_file( $override, $post_id, $meta_key, $meta_va
 	return update_pattern(
 		array_merge(
 			get_pattern_defaults(),
-			$pattern ? $pattern : [],
+			$pattern ? $pattern : [ 'title' => $post->post_title ],
```

Comment:
For a new pattern, `$pattern` will be false, as there won't be a pattern `.php` file yet.

So this should set the title from `$post->post_title`, instead of the `''` title that `get_pattern_defaults()` has.

Reply:
New patterns will have their titles [set in the WP_Post](https://github.com/studiopress/pattern-manager/blob/e14e9f6dfb78fd046d3f8871f488f9288f9b9ffe/wp-modules/editor/model.php#L230-L235), even though they won't be saved to a .php file yet.